online_store/gt_sport_v1.html

```diff
@@ -0,0 +1,120 @@
+ <!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="gt_sports.css">
+    <title>goodTeens sports online store</title>
+  </head>
+  <body>
+<h2 class="column" id="h2title">goodTeens - Sports</h2> <p>
+<h2 class="column" id="h2title2">Online Store</h2>
+<br><br>
+<p class="a"></p>
+<section class="card" id="product1"> 
+</section>
+<p class="b"></p>
+<section class="card" id="product2"> 
+</section>
+<p class="b"></p>
+<section class="card" id="product3"> 
+</section>
+<p class="b"></p>
+<section class="card" id="product4"> 
+</section>
+</section>
+<p class="b"></p>
+<section class="card" id="product5"> 
+</section>
+
+<script>
+const sportProducts = [
+  {
+    imgSrc: "skateboard.jpg",
+    name: "Skateboard",
+    brand: "Hudora",
+    level: "beginner",
+    price: 50,
+    seller: "Hudora",
+  },
+  {
+    imgSrc: "standuppaddle.jpg",
+    name: "Stand-up paddle",
+    brand: "Oceana",
+    level: "advanced",
+    price: 300,
+    seller: "Oceana",
+  },
+  {
+    imgSrc: "volleyball.jpg",
```

Comment:
I believe part of the homework was to use a nested object. Perhaps we could nest the image src and include alt text.

```js
{
    img: {
        source: "volleyball.jpg",
        altText: "volleyball image"
    },
   name: "Stand-up paddle",
...
```

Reply:
In my version_2, I have nested objects already, so unless this is necessary and requested, I wouldn't add another. Are you OK with that?